pages/en/lb4/Controllers.md

```diff
@@ -156,3 +156,52 @@ class HelloController {
 - A `Repository` is LoopBack's database abstraction. See [Repositories](Repositories.html) for more.
 - `@get('/messages')` creates the `Route` for the Operation using `app.route()`.
 - `@param.query.number` adds a `number` param with a source of `query`.
+
+## Handling Errors in Controllers
+
+In order to specify errors for controller methods to throw, the class `HttpErrors` is used. `HttpErrors` can be found in the `@loopback/rest` package.
```

Comment:
You can mention here if `HttpErrors` has an exhaustive list of http errors or a select ones which are common.

Reply:
+1 Would be helpful to have a list of all error's available (or atleast the common ones and a link to all the options in `http-errors` module).